src/cdk-experimental/ui-patterns/listbox/listbox.ts

```diff
@@ -91,6 +91,9 @@ export class ListboxPattern<V> {
     return this.inputs.textDirection() === 'rtl' ? 'ArrowLeft' : 'ArrowRight';
   });
 
+  /** Represents the space key. Does nothing when the user is actively using typeahead. */
+  spaceKey = computed(() => (this.typeahead.query().length ? '' : ' '));
```

Comment:
how about something like `spaceSelectionKey`, or something similar, to imply there's more to it than just watching `spaceKey`?

Below the logic just seems to always trigger on "space key"

Reply:
or should this just generally be `selectionKey` and include the 'Enter' key in this

Reply:
We wouldn't be able to include "Enter" unless we change the event manager to also accept Signal<Regex>. Imo the fact that we're using a signal instead of just the character " " already indicates there's more going on here, but I updated the name to "dynamicSpaceKey" anyway